package.json

```diff
@@ -49,17 +49,17 @@
     "glob": "^7.2.3",
     "jest": "^27",
     "jest-junit": "^13",
-    "jsii": "^1.70.0",
-    "jsii-diff": "^1.70.0",
+    "jsii": "^1.71.0",
+    "jsii-diff": "^1.71.0",
     "jsii-docgen": "^3.8.31",
-    "jsii-pacmak": "^1.70.0",
+    "jsii-pacmak": "^1.71.0",
     "json-schema": "^0.4.0",
     "npm-check-updates": "^16",
-    "prettier": "^2.7.1",
+    "prettier": "^2.8.0",
     "projen": "0.64.1",
```

Comment:
I think the latest version of Projen would solve this problem. Should we bump this version?